Added turing.com

KapilKumar7 opened the pull request:
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [ ] I am an employee of the company mentioned and confirm all included details are correct.
- [ ] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [ ] The company directly hires employees. No bootcamps / freelance sites / etc
- [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [ ] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [ ] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [ ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/tree/main/.github/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github/CODE_OF_CONDUCT.md).

dougaitken commented on company-profiles/turing.md:
```diff
@@ -0,0 +1,38 @@
+
+Worldwide
+
+## Company technologies
```
These aren't technologies so either remove this section completely or edit to match programming languages or similar.

KapilKumar7 replied:
I have removed the technologies section.

dougaitken commented on company-profiles/turing.md:
```diff
@@ -0,0 +1,38 @@
+
+## Remote status
+
+100 % remote jobs .
```
Two unneeded spaces. Please change to this:

```
100% remote jobs.
```

KapilKumar7 replied:
Done.

KapilKumar7 commented:
I have removed the whitespaces and Removed the technology section as well.  Please let me know if there is something else where I need to make changes.
